.github/actions/rust-setup/action.yaml

```diff
@@ -9,11 +9,6 @@ runs:
     - run: sudo apt-get update && sudo apt-get install build-essential ca-certificates clang curl git libpq-dev libssl-dev pkg-config lsof lld --no-install-recommends --assume-yes
       shell: bash
 
-    - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # pin@v1
```

Comment:
just remove with is enough.

Keep uses: <arg>

Reply:
Resolved.